docs/pages/reference/access-controls/roles.mdx

```diff
@@ -142,6 +144,52 @@ spec:
       'workload': ['database', 'backup']
 ```
 
+### `client_idle_timeout`
```

Comment:
I would use plain prose for this heading, rather than the config field name, since other headings use prose as well.